pkgs/development/interpreters/python/cpython/default.nix

```
@@ -214,7 +214,10 @@ in with passthru; stdenv.mkDerivation {
     substituteInPlace setup.py --replace /Library/Frameworks /no-such-path
   '';
 
-  patches = [
+  patches = optionals (version == "3.10.9") [
```

```suggestion
  patches = optionals (version == "3.10.9" || version == "3.11.1") [
```

Python 3.11 is also affected.

---

Ah, I thought it wasn't updated as I didn't see it in the same MR.  But this patch won't apply there, unfortunately.

---

3.11 at least isn't so common yet, so it should be way smaller rebuild and thus could be fixed later.  I expect we might simply `fetchpatch` https://github.com/python/cpython/commit/3fae04b10e2655a20a3aadb5e0d63e87206d0c67 separately for 3.11.

---

Fine, we can leave it as is and unblock just python310 right now.